### ISSUE TITLE
webdav sync issues

### ISSUE THREAD

**Issue by Crisr** [marcelklehr/floccus#450]
Can't use webdav synology server wit floccus

With synology webdav server created:
1. Enter webdav url with port in floccus options
2. Enter username/password
2. provide path to write bookmarks.xbel file
4. Press Sync Now button
All good! message appears but no bookmarks.xbel is to be found in webdav folder

**Expected behavior**
When "all good!" message appear when pressing the sync button, an actual sync should take place and bookmarks.xbel should be created.

**Desktop (please complete the following information):**
 - OS: Windows 10, OSX Catalina
 - Browser Firefox, Brave, Chrome
 - Browser Version: latest for all
 - Floccus version: 3.5.3
 - Floccus sync method: webdav

I've replaced the actual IP address with --localnasaddress--
**Debug log**
2020-02-13T07:43:34.416Z Starting sync process for account nas@--localnasaddress-- 
2020-02-13T07:43:34.422Z onSyncStart: begin
2020-02-13T07:43:34.423Z --localnasaddress--:5005/backup/bookmarks/bookmarks.xbel.lock
2020-02-13T07:43:34.496Z --localnasaddress--:5005/backup/bookmarks/bookmarks.xbel.lock
2020-02-13T07:43:34.532Z onSyncStart: completed
2020-02-13T07:43:34.534Z Using normal strategy
2020-02-13T07:43:34.646Z COMPARE { localItem: 
     + #E3VH7MTB3wXs[undefined] parentId: undefined, hash: undefined
   ,
  cacheItem: 
     + #undefined[undefined] parentId: undefined, hash: undefined
   ,
  serverItem: 
     + #0[root] parentId: undefined, hash: undefined
    }
2020-02-13T07:43:34.657Z Checking subtree
2020-02-13T07:43:34.782Z onSyncComplete
2020-02-13T07:43:34.784Z No changes to the server version necessary
2020-02-13T07:43:34.805Z Successfully ended sync process for account nas@--localnasaddress--

**Additional context**

1. I can access the remote webdav folders on both Mac OSX Catalina and Windows 10 (with winscp) 
2. I can create files and folders in the floccus specified path through webdav, using the same credentials as in floccus
3. when browsing the floccus folder on the nas I can see a "#Trash" folder containing several versions of the file "bookmarks.xbel.lock" each 42 bytes in length

**Comment by marcelklehr** [comment]
Hello @Crisr 
Which bookmarks folder did you select for syncing?

**Comment by Crisr** [comment]
Hello, this is how I set the folder:
![RT0E0W1](https://user-images.githubusercontent.com/1618835/74445644-62958280-4e7f-11ea-9fa1-a89066989e7f.png)

and it is indeed empty after sync.

However I think I've soved the issue.
The "Local folder" was set by default to some weird path (Nextcloud@my-ip or something like that.)
I've changed that to /Bookmarks Bar and boom, sync is working properly, bookmarks.xbel appeared in the correct folder.

Anyway, thanks for the great work.
